nerfstudio/exporter/exporter_utils.py

```diff
@@ -155,6 +155,13 @@ def generate_point_cloud(
                 normal = (normal * 2.0) - 1.0
             point = ray_bundle.origins + ray_bundle.directions * depth
 
+            # Filter points with opacity lower than 0.5
```

Comment:
It would be nice for this to be configurable

Reply:
Do you think it makes a lot of difference? I assume most opacity will be binary, with only a tiny fraction getting close to 0.5.

Reply:
This is an example histogram:
```
(array([28222,    51,    40,    26,    29,    27,    30,    46,    69,
        4228]), array([0.        , 0.10000004, 0.20000008, 0.3000001 , 0.40000015,
       0.5000002 , 0.6000002 , 0.7000002 , 0.8000003 , 0.90000033,
       1.0000004 ], dtype=float32))
```

opacity is indeed mainly binary. Let's keep the hardcoded threshold :)

Reply:
Ok